src/build/mod.rs

```diff
@@ -0,0 +1,38 @@
+use crate::{
+    external_cli::{cargo, rustup, wasm_bindgen},
+    manifest::package_name,
+};
+
+pub use self::args::BuildArgs;
+
+mod args;
+
+pub fn build(args: &BuildArgs) -> anyhow::Result<()> {
+    if args.is_web() {
+        ensure_web_setup()?;
+    }
+
+    let cargo_args = args.cargo_args_builder();
+
+    if args.is_web() {
```

Comment:
```suggestion
    let cargo_args = args.cargo_args_builder();

    if args.is_web() {
        ensure_web_setup()?;
```

`ensure_web_setup()` can be moved down so that we don't need to check `args.is_web()` twice.